### DISCUSSION TITLE
Changing columns map reference after data

### DISCUSSION THREAD

**Discussion by vitolpoc** [infinite-table/infinite-react#6] [Q&A] [answered]
Hi,

When setting (or changing) column definition map after data, nothing is displayed.

Here is a sandbox link that reproduce the issue: https://codesandbox.io/s/sandpack-project-forked-hk070

**Comment by roblotter** [comment, marked as answer]
Thanks for the feedback. This issue should now be fixed in version `0.0.7`

See https://codesandbox.io/s/sandpack-project-forked-qh010?file=/src/App.tsx

**Reply by roblotter** [reply]
You can also see it on the releases page here https://infinite-table.com/docs/releases#007-

**Reply by roblotter** [reply]
The related issue to this discussion is #7
